tools/ossf_best_practices/silver_criteria.md

```diff
@@ -0,0 +1,321 @@
+
+_Possible answers: Met/Unmet/NA_
+
+
+## Installation system
+
+> The project MUST provide a way to easily install and uninstall the software produced by the project using a commonly-used convention.
+
+_Possible answers: Met/Unmet/NA_
+
+> The installation system for end-users MUST honor standard conventions for selecting the location where built artifacts are written to at installation time. For example, if it installs files on a POSIX system it MUST honor the DESTDIR environment variable. If there is no installation system or no standard convention, select "not applicable" (N/A).
+
+_Possible answers: Met/Unmet/NA_
+
+> The project MUST provide a way for potential developers to quickly install all the project results and support environment necessary to make changes, including the tests and test environment. This MUST be performed with a commonly-used convention.
+
+_Possible answers: Met/Unmet/NA_
+
+
+
+## Externally-maintained components
+
+> The project MUST list external dependencies in a computer-processable way. (URL required)
+
+_Possible answers: Met/Unmet/NA_
+
+> Projects MUST monitor or periodically check their external dependencies (including convenience copies) to detect known vulnerabilities, and fix exploitable vulnerabilities or verify them as unexploitable.
+
+_Possible answers: Met/Unmet/NA_
+
+> The project MUST either:
+>   1. make it easy to identify and update reused externally-maintained components; or
+>   2. use the standard components provided by the system or programming language.
+> 
+> Then, if a vulnerability is found in a reused component, it will be easy to update that component.
+
+_Possible answers: Met/Unmet/NA_
+
+
+> The project SHOULD avoid using deprecated or obsolete functions and APIs where FLOSS alternatives are available in the set of technology it uses (its "technology stack") and to a supermajority of the users the project supports (so that users have ready access to the alternative).
+
+_Possible answers: Met/Unmet/NA_
+
+
+## Automated test suite
+
+> An automated test suite MUST be applied on each check-in to a shared repository for at least one branch. This test suite MUST produce a report on test success or failure.
+
+_Possible answers: Met/Unmet_
+
+> The project MUST add regression tests to an automated test suite for at least 50% of the bugs fixed within the last six months.
+
+_Possible answers: Met/Unmet/NA_
+
+> The project MUST have FLOSS automated test suite(s) that provide at least 80% statement coverage if there is at least one FLOSS tool that can measure this criterion in the selected language.
+
+_Possible answers: Met/Unmet/NA_
+
+
+## New functionality testing
+
+> The project MUST have a formal written policy that as major new functionality is added, tests for the new functionality MUST be added to an automated test suite.
+
+_Possible answers: Met/Unmet/NA_
```

Comment:
```suggestion
> The project MUST have a formal written policy that as major new functionality is added, tests for the new functionality MUST be added to an automated test suite.

**Met**
```

Reply:
Met

Reply:
Resolved in 4c286d63e1bfb1a1d3688f2514b37042a3d1d082